extensions/spark/kyuubi-spark-connector-hive/pom.xml

```diff
@@ -0,0 +1,218 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one or more
+  ~ contributor license agreements.  See the NOTICE file distributed with
+  ~ this work for additional information regarding copyright ownership.
+  ~ The ASF licenses this file to You under the Apache License, Version 2.0
+  ~ (the "License"); you may not use this file except in compliance with
+  ~ the License.  You may obtain a copy of the License at
+  ~
+  ~    http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <groupId>org.apache.kyuubi</groupId>
+        <artifactId>kyuubi-parent</artifactId>
+        <version>1.6.0-SNAPSHOT</version>
```

Comment:
This feature is exptect for 1.7, branch-1.6 has been cut, and the master branch has go ahead to 1.7.0-SNAPSHOT, please rebease your PR on master and change the version.

BTW, do you want to support old Spark versions, or just for 3.3 and above?

Reply:
same concern about spark version, because support of some basic commands diffently amoung spark 3.1/3.2/3.3, ex: `show create table` only works >=3.2, `set/use catalog` only works >=3.3,  if we consider downward compatibility, do we need to  sync up this capabilities?

Reply:
consent is supported only in version 3.3 or above, as in https://github.com/apache/incubator-kyuubi/issues/3259#issuecomment-1219243969